airbyte-integrations/connectors/destination-milvus/pyproject.toml

```
@@ -16,9 +16,13 @@ repository = "https://github.com/airbytehq/airbyte"
 [[tool.poetry.packages]]
 include = "destination_milvus"
 
+[tool.poetry.dependencies.airbyte-cdk]
+# Version cnstrained by PyAirbyte (`airbyte`) version
+version = ">=1.0"
```

What happens if we try to take the latest CDK version here?

```suggestion
version = "^6.9.0"
```

---

have updated the CDK version and had to update the python version